packages/react/src/products/hooks/useProductGroupingProperties.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  areProductGroupingPropertiesFetched,
+  areProductGroupingPropertiesLoading,
+  buildQueryStringFromObject,
+  fetchProductGroupingProperties,
+  getProductGroupingProperties,
+  getProductGroupingPropertiesError,
+  ProductEntity,
+  StoreState,
+} from '@farfetch/blackout-redux';
+import { useAction } from '../../helpers';
+import { useCallback, useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import type { UseProductGroupingPropertiesOptions } from './types';
+
+const useProductGroupingProperties = (
+  productId: ProductEntity['id'],
+  options: UseProductGroupingPropertiesOptions = {},
+) => {
+  const { fetchConfig, enableAutoFetch = true, fetchQuery = {} } = options;
```

Comment:
Does `fetchQuery` needs to have a default value here? I think as the endpoint does not require it, you can remove the default value and avoid having the `fetchQuery` having a different object reference for each render, which might trigger the useEffect for fetching as it will create a new `fetch` reference each time the hook is rendered.